### PR TITLE
Add a line about updating theme-gems in the docs

### PR DESCRIPTION
**Pull request by ashmaroli**
Update documentation to add a line about publishing a new version of theme gems to help those who may not know how.

@jekyll/documentation
@DirtyF

### CODE REVIEW

File: site/_docs/themes.md [hunk 1]
```diff
@@ -94,3 +94,5 @@ Themes are published via [RubyGems.org](https://rubygems.org). You'll need a Rub
 2. Next, push your packaged theme up to the RubyGems service, by running the following command, again replacing `my-awesome-jekyll-theme` with the name of your theme:
 
         gem push my-awesome-jekyll-theme-*.gem
+
+3. To release a new version of your theme, simply update the version number in the gemspec file, ( `my-awesome-jekyll-theme.gemspec` in this example ), following the principles of [Semantic Versioning](http://semver.org/) and then repeat Steps 1 & 2 above.
```
Comment (DirtyF) [inline site/_docs/themes.md]: let's be specific like in the above steps.
To release 
* bump version in `spec.version       = "0.1.0"` in your `.gemspec`

**Comment by DirtyF** [comment]
Thanks @ashmaroli, I saw the question popped on Jekyll Talk, good call.

LGTM.

**Comment by envygeeks** [comment]
I think the line about SemVer needs to be less authoritative and more suggestive. Just my 0.02c

**Comment by parkr** [comment]
LGTM. Thanks!

@jekyllbot: merge +site

**Pull request merged**